.rusty-hook.toml

```
@@ -0,0 +1,5 @@
+[hooks]
+pre-commit = "just all"
```

I often do few commits before pushing. Is that's enough for pre-commit?
```suggestion
pre-commit = "just lint test"
pre-push = "just all"
```